### ISSUE TITLE
Adding an example to the enumeration can be nice

### ISSUE THREAD

**Issue by QDucasse** [SquareBracketAssociates/Booklet-uFFI#15]
I feel like a quick example at the end of chapter 3.4, once the poolDictionary is added can be a nice conclusion to the chapter!

**Comment by QDucasse** [comment]
4.4 now sorry

**Comment by linkerstorm** [comment]
Yes, an example would be nice, because in my uFFI package for [FreeTDS](https://www.freetds.org/), trying to implement poolDictionaries the way described in the document failed for me.
I certainly miss something...

**Comment by DiagProf** [comment]
Hi linkerstorm 
> I certainly miss something...
I think I struggled across the same problem.
In the document you can read ..... " defining a
subclass of FFIEnumeration, a enumDecl class-side method returning the
specification of the enumeration elements, and finally sending the initialize message to the enumeration class we created."

and 

"Doing this will automatically generate some boilerplate code to manipulate
the enumeration. You will see that the enumeration class gets redefined as
follows creating and initializing a class variable for each of its elements."

But when I look at FFITestEnumeration class  (Inside Pharo). I can see that there are access methods.
e.g.  AAA
	"This method was automatically generated"
	^ AAA

And there is in the comment : "This method was automatically generated"

Now I don't know if there is anything wrong in the book or if Pharo is making something wrong. 
But if you have these methods it also works with the "poolDictionaries"
